addons/voice/org.openhab.voice.gtts/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+		http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="voice:gtts">
+		<parameter name="serviceAccountKeyName" type="text" required="true">
+			<label>Service Account Key Name</label>
+			<description>Google Cloude TTS API service account key name. The addon will try to load this file from $USER_DATA/gtts folder.</description>
```

Comment:
typo: Cloude

Reply:
Actually, user configuration should rather be in the conf and not the userdata folder - userdata is the one that is written by the system and which might be (partially) deleted when doing an update. I do not have a very good suggestion for a folder in conf though.

In any case, ideally the configuration should be stored within the service itself, i.e. this parameter would simply take the json content that the user pastes in. Is there any strong reason against doing it that way?

Reply:
Configuration changed

Reply:
BTW the idea of storing the key in userdata was based on examples seen in other bindings: e.g. openhabcloud stores the secret also in a file under $userdata/openhabcloud/secret. Is there any difference between the two components?

Reply:
Yes, the difference is that the openhabcloud secret is generated by openHAB and thus it is written to userdata (which is the only writable folder it has access to).

Reply:
The summary on the bottom of the page says that you requested some changes. I thing I addressed them all. Is there anything else I can do? Thx